### ISSUE TITLE
Weird Expected Output Test Case 8_4

### ISSUE THREAD

**Issue by gordengorden** [nus-cs2030/2122-s2#201]
# Summary
Encountered a failed test case today while working on level 8, however I'm not sure how to interpret the expected output.

## Description 
From the question

> When we print out the wait event, we always say that the customer is waiting for the self-checkout counter k + 1, even though this customer may eventually be served by another self-checkout counter.

However, the test case shows not only customers waiting at a self-checkout counter with different k + 1, but also with the id number of a human server.

## Screenshots (if any): 
![image](https://user-images.githubusercontent.com/46585440/161367200-a33ef236-65de-4474-a448-ff4252e31373.png)

**Comment by Yang990121** [comment]
Did you manage to solve this? I have the exact same issue :(

**Comment by Yang990121** [comment]
This is what im getting, similar to yours 

<img width="598" alt="Screenshot 2022-04-02 at 8 33 24 PM" src="https://user-images.githubusercontent.com/65878741/161383705-1945a463-2b02-4cb7-990b-74ab7f6161cb.png">

**Comment by Yang990121** [comment]
This is my results, i really think its a typo :(
<img width="602" alt="image" src="https://user-images.githubusercontent.com/65878741/161385040-3b0342ec-b79d-4557-94e5-2bbd7ff91e85.png">

**Comment by gordengorden** [comment]
> Did you manage to solve this? I have the similar issue :(

Nope not yet, although I did manage to find a tiny mistake in my program, edited with my most recent test output. Maybe we can try reverse engineering the test case to understand what's the underlying issue? Looks like its incrementing k+1 for each customer in the queue.

**Comment by jingyulim1** [comment]
From the question
> There are now Nself self-checkout counters set up. In particular, if there are k human servers, then the self-checkout counters are identified from k + 1 onwards.

It should be a typo as both 1 and self-check 1 exists (line 28 and line 30).

We also see in test8_2 that the the first server is self-check3 when there is 2 human servers.

**Comment by hwkchia** [comment]
Yes, you are right.  There should not be a self-check 1.  I've updated the test case.

Apologies for having caused any misunderstanding.

**Comment by jingyulim1** [comment]
Works now, thanks Prof!

**Comment by Yang990121** [comment]
Thank you prof for the speedy reply on a late Saturday night :D

**Comment by hwkchia** [comment]
To show my appreciation for your hard work despite the weekend, I've included the fourth test case in the question. 😄
